documentation/source/quickstart.rst

```diff
@@ -4,291 +4,188 @@
 Quickstart Guide
 ****************
 
+In the following sections,
+we are walking you through creating and running a small but complete cocotb testbench
+for a fictional *Design Under Test* (:term:`DUT`) called ``my_design``.
+We assume the DUT is available in both a Verilog and a VHDL implementation.
 
-Running your first Example
-==========================
-
-Make sure you have the :ref:`prerequisites<install-prerequisites>`
+Please install the :ref:`prerequisites<install-prerequisites>`
 (Python with development packages, a C++11 compiler with development packages, GNU Make,
```

Comment:
I'd remove this part, it's very easy for those two pages to go out of sync and instead add a small check like "Run ``cocotb-config --version`` in a terminal window to check if cocotb is installed."